src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -179,6 +138,95 @@ class AadO365GroupAddCommand extends GraphCommand {
       }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, logger, cb));
   }
 
+
+  private validateOwners(logger: Logger, args: CommandArgs): Promise<any> {
```

Comment:
Let's not use `any` as the return type. As we expect a `string[]` let's set that as the method's return type to avoid runtime errors in the future.